docs/rst/tutorials/cloud/wan_edge_cloud/wan_edge_cloud.rst

```diff
@@ -0,0 +1,280 @@
+
+.. literalinclude:: /resources/tutorials/cloud/wan_edge_cloud/dds_router_edge.yaml
+    :language: yaml
+
+Next, it is briefly explained the most relevant aspects of this configuration file.
+
+The ``participants`` are the interfaces of the DDS Router to communicate with other networks. In this case, we have two participants:
+
+    *   ``local``: this is a simple participant that communicates with all ROS 2 nodes it finds.
+        For more information about this participant please refer to the `Simple Participant section <https://eprosima-dds-router.readthedocs.io/en/latest/rst/user_manual/participants/simple.html#user-manual-participants-simple>`_ of the DDS Router documentation.
+
+    *   ``router``: it is a participant designed for the communication between two *DDS Routers*.
+        It uses the `Fast DDS Discovery Server discovery mechanism <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/discovery_server.html#discovery-server>`_ to establish a point-to-point communication between two DDS entities, two *DDS Routers* in this case.
+        It is not necessary to know more about this discovery protocol, but it is important to note that each ``router``-type participant must have a unique ``id`` across the entire network, which is configured with the ``discovery-server-guid`` tag.
+
+    For the DDS Router Edge, a connection address shall be defined which must be the same as the one exposed by the Cloud Server.
+    There are some relevant configurations within this connection address:
+
+    * ``discovery-server-guid``: it defines the ``id`` of the ``router``-type participant of the DDS Router Cloud.
+    * ``addresses``: defines the IP (``ip``) and port (``port``) of the network addresses to which it connects, and the transport protocol (``transport``) to be used in the communication, TCP in this case.
+
+.. note::
+
+    It is important to mention that the ROS 2 topics relayed by the DDS Router can be filtered by configuring the ``allowlist`` and ``blocklist``.
+    However, in this case the DDS Router will forward all topics found in the network, so this is not configured.
+    In case this is the case please refer to the `DDS Router documentation <https://eprosima-dds-router.readthedocs.io/en/latest/>`_ for information on how to do this.
+
+The following figure summarizes the deployment on the edge.
+
+.. figure:: /rst/figures/tutorials/cloud/edge_deployment.png
+   :align: center
+
+To finish this step, run the DDS Router with the configuration file created as an argument.
+
+.. code-block::
+
+    ddsrouter -c <path/to/file>/ddsrouter_edge.yaml
+
+Running the turtlesim_square_move on the Cloud
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Run the ``turtlesim_square_move`` in the Cloud Server machine, which is the controller of the edge ``turtlesim_node``.
+This will send commands to the ROS 2 application on the edge to move the turtle and receive information about the current state of the turtle at any time.
```

Comment:
```suggestion
This will send commands to the ROS 2 application to the edge to move the turtle and receive information about the current state of the turtle at any time.
```